infrastructure/database/migrations/7_introducing_identifier_column_workflow.up.sql

```diff
@@ -0,0 +1,5 @@
+ALTER TABLE workflows ADD COLUMN 'identifier' TEXT;
```

Comment:
I think it would be best if there would be a unique index on `identifier`, so that we guarantee there are no two workflows with the same identifier. To do so however, I'm assuming the column will need to be of a type other than `TEXT` (.e.g `VARCHAR(N)`), which I think would also make sense, since I believe `TEXT` is intended for "long" text fields.

Reply:
Oh yes, we discussed that earlier. I will add a UNIQUE constraint on the column. SQLite actually can store whatever data in whatever column. And natively TEXT is what it supports as one of the type for strings, which is good enough for the time being. It does have a concept of text affinity so that queries are compatible with more database types, but I don't think we would want to worry about that at all for now.

Reply:
Altering a column isn't possible in SQLite so we have got insert the entire table into another table with actual columns we want and then rename that table. But I can't seem to make a query like this work:

```
INSERT INTO "workflows_tmp" (id,name,description,active,identifier)
SELECT
  id,
  name,
  description,
  active,
  (SELECT value FROM workflow_meta WHERE key = 'toml_identifier' AND 'workflow_id' = id)
FROM
  "workflows";
```

Reply:
Never mind, figured it out!

Reply:
@psrpinto Back to you for review!